source/containers.tex

```diff
@@ -17357,11 +17357,11 @@
 
   template<class InputIterator, class Compare = less<@\placeholder{iter-value-type}@<InputIterator>>>
     flat_multiset(InputIterator, InputIterator, Compare = Compare())
-      -> flat_multiset<@\placeholder{iter-value-type}@<InputIterator>, @\placeholder{iter-value-type}@<InputIterator>, Compare>;
+      -> flat_multiset<@\placeholder{iter-value-type}@<InputIterator>, Compare>;
 
   template<class InputIterator, class Compare = less<@\placeholder{iter-value-type}@<InputIterator>>>
     flat_multiset(sorted_equivalent_t, InputIterator, InputIterator, Compare = Compare())
-      -> flat_multiset<@\placeholder{iter-value-type}@<InputIterator>, @\placeholder{iter-value-type}@<InputIterator>, Compare>;
+      -> flat_multiset<@\placeholder{iter-value-type}@<InputIterator>, Compare>;
```

Comment:
This corrects (what to me is) obvious copy-pasta from `flat_multimap`. I call it "editorial" despite that it's a borderline normative change.